### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by m-yahya**

### CODE REVIEW

File: build.Dockerfile [hunk 1]
```diff
@@ -18,8 +18,10 @@
 
 ### Cached Builder Stage
 ##################################################
-# For the OLI guys, we removed the sccache part is it makes the native GA runners run out of space.
```
Comment (clangenb) [inline build.Dockerfile:21]: Please leave the comment I left for you guys. 😄 hence, you can remove the sccache installation from below.

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -1,7 +1,6 @@
 services:
   "integritee-node-${VERSION}":
-    # OLI: stripped `-ias` from node name.
```
Comment (clangenb) [inline docker/docker-compose.yml:3]: This can indeed be removed now 👍

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -44,8 +43,7 @@ services:
       interval: 10s
       timeout: 10s
       retries: 25
-    # OLI: added `--skip-ra`
```
Comment (clangenb) [inline docker/docker-compose.yml:47]: Guys need the skip-ra. 😊

File: docker/docker-compose.yml [hunk 4]
```diff
@@ -74,8 +72,7 @@ services:
...
```
Comment (clangenb) [inline docker/docker-compose.yml:77]: Keep the skip-ra

**Review by clangenb — COMMENTED** [review]
There are indeed tiny fixes tjat you need to do, and then it should work.

File: build.Dockerfile (hunk 1 above)
Comment (m-yahya) [reply build.Dockerfile:21]: I'll keep the comment.

File: docker/docker-compose.yml (hunk 2 above)
Comment (m-yahya) [reply docker/docker-compose.yml:3]: Removed.

File: docker/docker-compose.yml (hunk 3 above)
Comment (m-yahya) [reply docker/docker-compose.yml:47]: flag added

File: docker/docker-compose.yml (hunk 4 above)
Comment (m-yahya) [reply docker/docker-compose.yml:77]: flag added

**Pull request merged**